### PR TITLE
Definition for use function

### LINKED ISSUE #94: Definition for use function
```php
use function LanguageServer\pathToUri;
```
Cannot get definition or hover for `pathToUri`

### PR DESCRIPTION
**Pull request by mniewrzal**
Closes #94

### CODE REVIEW

File: src/Fqn.php [hunk 1]
```diff
@@ -45,6 +45,10 @@ function getReferencedFqn(Node $node)
         $grandParent = $parent->getAttribute('parentNode');
         if ($grandParent instanceof Node\Stmt\GroupUse) {
             $name = $grandParent->prefix . '\\' . $name;
+        } else if ($grandParent instanceof Node\Stmt\Use_) {
+            if ($grandParent->type === Node\Stmt\Use_::TYPE_FUNCTION) {
```
Comment (felixfbecker) [inline src/Fqn.php]: Use a `&&`

**Comment by mniewrzal** [comment]
> I assume use const is not affected because the FQN does not need ()

Yes :)

**Comment by felixfbecker** [comment]
Btw you need to link the issue in the description, not the title so GitHub can close it

**Comment by mniewrzal** [comment]
ah, ok :) next time

**Pull request merged**